tests/xharness/AppRunner.cs

```diff
@@ -655,26 +659,27 @@ public async Task<int> RunAsync ()
 				} else {
 					args.Add ("--wait-for-exit");
 				}
-				
+
 				AddDeviceName (args);
 
 				var deviceSystemLog = Logs.Create ($"device-{deviceName}-{Helpers.Timestamp}.log", "Device log");
-				var deviceLogCapturer = deviceLogCapturerFactory.Create (harness.HarnessLog,deviceSystemLog, deviceName);
+				var deviceLogCapturer = deviceLogCapturerFactory.Create (harness.HarnessLog, deviceSystemLog, deviceName);
 				deviceLogCapturer.StartCapture ();
 
 				try {
-					await crash_reports.StartCaptureAsync ();
+					await crashReporter.StartCaptureAsync ();
 
 					MainLog.WriteLine ("Starting test run");
 
 					bool waitedForExit = true;
 					// We need to check for MT1111 (which means that mlaunch won't wait for the app to exit).
 					var callbackLog = new CallbackLog ((line) => {
 						// MT1111: Application launched successfully, but it's not possible to wait for the app to exit as requested because it's not possible to detect app termination when launching using gdbserver
-						waitedForExit &= line?.Contains ("MT1111: ") != true;
+						waitedForExit &= !line?.Contains ("MT1111: ") != true;
```

Comment:
This looks strange, what's the reason for this change?

Reply:
Oops, my bad, will revert